src/test/java/org/cactoos/io/TeeInputStreamTest.java

```diff
@@ -79,7 +79,7 @@ public void leftInputUnclosed() {
             MatcherAssert.assertThat(
                 "Can't use output after usage from TeeInput",
                 write.isClosed(),
-                new IsEqual<>(false)
+                new IsEqual<>(true)
```

Comment:
@aivinog1 please rename this test accordingly

Reply:
Done